src/types/editor-version.ts

```diff
@@ -1,40 +1,77 @@
-/**
- * Describes a version of a Unity editor. Mostly this follows calendar-versioning,
- * with some extra rules for chinese releases.
- * @see https://calver.org/
- */
-export type EditorVersion = {
+type LocaleCode = "c";
+
+type ReleaseFlag = "a" | "b" | "f" | "c";
+
+type RegularVersion = {
   /**
    * The major version. This is the release year.
    */
   major: number;
   /**
    * The minor version. This is usually a number from 1 to 3.
    */
-  minor: number;
+  minor: 1 | 2 | 3 | 4;
```

Comment:
I suggest we leave the minor field as a number to be prepared for the new version schema (Unity 6.x)

Reply:
Makes sense. I reverted it.